GitCommands/Settings/AppSettings.cs

```diff
@@ -1400,6 +1400,24 @@ public static bool RememberNumberOfContextLines
             set => SetBool("RememberNumberOfContextLines", value);
         }
 
+        public static bool ShowSyntaxHighlightingInDiff
+        {
+            get => RememberShowSyntaxHighlightingInDiff && GetBool("ShowSyntaxHighlightingInDiff", true);
```

Comment:
I disagree with this, these are settings - i.e. what a user has chosen. We should not have conditions here.
How settings combined affect the UI should be done at the UI layer.

Reply:
The implementation follows the existing options, so I  believe this is fine.
However, I suggest the default for RememberShowSyntaxHighlightingInDiff  is changed to true, which would be about the same.